file.js

```diff
@@ -1,83 +1,107 @@
+  },
+  {
+    name: 'Martin Bah',
+    profession: 'Senoir Developer at Microsoft',
+    Image: 'Images/speaker03.png',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad',
+  },
+  {
+    name: 'David Ngong',
+    profession: 'Professor of Information Technology at Harvard University',
+    Image: 'Images/speaker04.png',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad',
+  },
+  {
+    name: 'Milton Njonya',
+    profession: 'CEO and Founder of  WEBDEV MENTORS',
+    Image: 'Images/speaker05.png',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad',
+  },
+  {
+    name: 'Milton Njonya',
+    profession: 'CEO and Founder of  WEBDEV MENTORS',
+    Image: 'Images/speaker06.png',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad',
+  },
+];
 
 const btnMore = document.getElementById('btn-more');
 const btnHide = document.getElementById('btn-hide');
-document.querySelector('.speaker').innerHTML = data.map((item, index)=>{
-    let className = '';
-   if(index !== 0 && index !== 1){
-     className = 'hide-card';
-   }
+document.querySelector('.speaker').innerHTML = data.map((item, index) => {
+  let className = '';
+  if (index !== 0 && index !== 1) {
+    className = 'hide-card';
+  }
 
-   btnMore.addEventListener('click', ()=>{
-   const hideCard = document.querySelectorAll('.hide-card');
-   for(i = 0; i< hideCard.length; i++){
-    hideCard[i].style.display = 'flex';
-   }
-   btnHide.style.display = 'block';
-   btnMore.style.display = 'none'
-   })
+  btnMore.addEventListener('click', () => {
+    const hideCard = document.querySelectorAll('.hide-card');
+    for (let i = 0; i < hideCard.length; i++) {
+      hideCard[i].style.display = 'flex';
+    }
+    btnHide.style.display = 'block';
+    btnMore.style.display = 'none';
+  });
 
-   btnHide.addEventListener('click', ()=>{
+  btnHide.addEventListener('click', () => {
     const hideCard = document.querySelectorAll('.hide-card');
-    for(i = 0; i< hideCard.length; i++){
-     hideCard[i].style.display = 'none';
+    for (let i = 0; i < hideCard.length; i++) {
+      hideCard[i].style.display = 'none';
     }
     btnHide.style.display = 'none';
-    btnMore.style.display = 'block'
-    })
-    return `
+    btnMore.style.display = 'block';
+  });
+  return `
+    <div class = "container">
        <div class = "speaker-card ${className}">
          <div class="image-container">
                 <img src="${item.Image}" alt="speaker" />
```

Comment:
- [OPTIONAL] You have done well generating this section dynamically as required. :+1: :+1: But looking at your project and the screenshot below, you can see you have some extra characters like `comas` which are not together with the text and are not designs also. These characters are not in the design, I'll suggest you go through your code and remove these. :+1: :+1: 


- - Screenshots:

![Selection_079](https://github.com/NalianAchei2020/Capstone/assets/107632780/28ce80a7-f0ea-4fbd-a306-6ee52c55dd6e)

![Selection_080](https://github.com/NalianAchei2020/Capstone/assets/107632780/187b1e2a-73a3-487e-ad38-d3e5ff01fb86)